backend/imports/tasks/methods.js

```diff
@@ -51,11 +51,11 @@ export const insertTask = function ({ text }) {
  * @param checked {boolean}
  * @return {number} 1 if successfull, otherwise 0
  */
-export const checkTask = function ({ _id, checked }) {
-  const userId = this.userId
-  checkUser(userId)
-  return TasksCollection.update({ _id, userId }, { $set: { checked } })
-}
+export const checkTask = async function ({ _id, checked }) {
+  const userId = this.userId;
+  checkUser(userId);
+  return await TasksCollection.update({ _id, userId }, { $set: { checked } });
```

Comment:
shouldn't it be `await TasksCollection.updateAsync` ?

Reply:
Yes you are right